src/app/business/routes/model/sagas/index.js

```diff
@@ -45,17 +35,9 @@ function* fetchDetail({payload}) {
     }
 }
 
-
 function* fetchPersistent(request) {
     const state = yield select();
-    let jwt;
-
-    if (typeof window !== 'undefined') {
-        const cookies = cookie.parse(window.document.cookie);
-        if (cookies['header.payload']) {
-            jwt = cookies['header.payload'];
-        }
-    }
+    const jwt = getJWTFromCookie();
```

Comment:
As above, this looks like an incomplete refacto.

Reply:
yes same thing, don't know if I should use tryRefreshToken here